pikaday-directive.js

```diff
@@ -29,6 +29,20 @@ angular.module('pikaday', [])
                 }
             };
 
+            // Adds weekday/weekend class to cells
+            // Pikaday hides render implementation, need to consider forking it
```

Comment:
If we fork Pikaday, how will we handle other packages that depend on it? Will we have to fork them, too? Is the Pikaday project open to pull requests? Please find out.